docs/source/modin/supported/groupby_supported.rst

```diff
@@ -106,7 +106,8 @@ Computations/descriptive stats
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``ffill``                   | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``fillna``                  | N                               |                                                    |
+| ``fillna``                  | P                               | GroupBy axis = 0 is supported.                     |
```

Comment:
It seems axis=1 is also supported?

Reply:
No, the general case of groupby axis=1 is not supported all the df based on the docstrings.

From dataframe_supported.rst:
![Screenshot 2024-09-23 at 4 10 42 PM](https://github.com/user-attachments/assets/13a124e7-85e5-4dee-83c9-b216b80431cd)